tests/zenko_tests/node_tests/utils/request.js

```diff
@@ -36,12 +36,12 @@ function getResponseBody(res, cb) {
  * @param {Function} cb - callback(error, response)
  * @return {undefined}
  */
-function makeGETRequest(path, cb) {
+function makeGETRequest(path, cb, userCredentials) {
     let options = Object.assign({}, defaultOptions, {
         method: 'GET',
         path,
     });
-    options = aws4.sign(options, credentials);
+    options = aws4.sign(options, userCredentials ? userCredentials : credentials);
```

Comment:
`userCredentials || credentials` ?

Reply:
this would return true or false
But I assume you wanted to say `userCredentials | credentials`, I will change this :)

Reply:
If the first variable is null or undefined, it will return the second variable:

```
undefined || 'test'
> 'test'
```